### DISCUSSION TITLE
2008/01/14/%E6%9C%89%E9%81%93%E7%94%B5%E5%AD%90%E8%AF%8D%E5%85%B8

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#206] [Announcements]
# 2008/01/14/%E6%9C%89%E9%81%93%E7%94%B5%E5%AD%90%E8%AF%8D%E5%85%B8

由于英语是工作语言，而我的英语不是非常好，所以电子词典就成了我电脑上必备的软件。相当长一段时间，我一直使用的是金山词霸。但是，我对它的翻译并不是太满意。词典的一大弊病是翻译的很不自然，经典的词典多少年都更新不了一次，很难跟上日常用语的变化。往往词典上给出的翻译不是平时最常用的那个词，而是一个比较晦涩的生僻词。

https://blog.qizhen.xyz/2008/01/14/%E6%9C%89%E9%81%93%E7%94%B5%E5%AD%90%E8%AF%8D%E5%85%B8

**Comment by ruanqizhen** [comment]
之前博客下的留言：

zhancang
2008年02月2日 10:08 编辑
哥们儿,你好,我也是LabVIEW爱好者,有时间交流,我也会在Space上写些体会的,呵呵

回复
ωｪййｪё
2008年01月30日 03:51 编辑
字典有時候解釋的很奇怪，而且很多句子根本就翻譯的完全不對

回复
uni©ute™
2008年01月21日 12:09 编辑
想到以前读书的时候买过2个电子词典,结果都被偷了…

回复
zhang
2008年01月18日 18:27 编辑
留言是空的？什么意思？呵呵，似乎我们是同行，都是搞计算机的

回复
Woo
2008年01月18日 11:41 编辑
去试试,谢谢
 

回复
HA
2008年01月16日 15:20 编辑
偶也不是太喜欢金山-_-!
试试这个看看阿灵，呵呵。

回复
Nicholas
2008年01月15日 21:51 编辑
好东东，正愁家里的机器没字典，就来了一个:)

回复
huiyan
2008年01月14日 23:29 编辑
我一直使用Lingoes 灵格斯词霸,感觉不错,建议大侠试一试.
另外,我一直在关注你的博客,你写的有关labview的文章很深刻,如果以后有什么难题还望大侠赐教~~

回复
竹贤
2008年01月14日 21:46 编辑
不错。确实比金山词霸等电子词典似乎方便一点。只是不知在专业词汇方面如何，必须用用试试。

回复
杰
2008年01月14日 19:55 编辑
这个要试试看，哈哈。

回复
